src/main/java/sirius/db/mixing/properties/MultiLanguageStringProperty.java

```diff
@@ -97,6 +99,30 @@ protected void onBeforeSaveChecks(Object entity) {
                                                    .handle();
                                });
         }
+
+        Object propertyValue = getValue(entity);
+        boolean withFallback = multiLanguageString.isWithFallback();
+        Map<String, String> values = ((Map<String, String>) propertyValue);
+
+        if (!isNullable()) {
+            if(withFallback && Strings.isEmpty(values.get(MultiLanguageString.FALLBACK_KEY))) {
```

Comment:
please run code format before creating a PR - nothing worse than merging code style changes. and please invoke maven at least once to have proper code style setting in place.

Reply:
Thanks you the hint - i'll recheck my settings and do a mvn clean compile again to get the idea codeformat settings.